pkg/apis/addons/v1alpha1/common.go

```diff
@@ -53,7 +53,7 @@ type CommonAddonsConfigurationSpec struct {
 	Repositories     []SpecRepository `json:"repositories"`
 }
 
-// Addon holds information about single addon
+// AddonWithCharts holds information about single addon
 type Addon struct {
```

Comment:
Why do you call this `AddonWithCharts`?

Reply:
yes, here and in the storage package the IntelliJ renamed too much. I will revert that